src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DoNotDefineFinalizersForTypesDerivedFromMemoryManagerTests.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Comment:
If this analyzer runs on generated code, it will need to use `CSharpSecurityCodeFixVerifier` (which doesn't validate that generated code is excluded).

```suggestion
using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
```